packages/render/src/identity/farcaster/use-farcaster-multi-identity.tsx

```diff
@@ -314,20 +321,20 @@ export function useFarcasterMultiIdentity({
                 key: keypairString.publicKey,
                 signature,
                 requestFid,
+                sponsorship,
                 deadline,
               }),
             })
           ).json()) as {
             result: {
-              signedKeyRequest: { token: string; deeplinkUrl: string };
+              signedKeyRequest: { token: string; deeplinkUrl: string, isSponsored: boolean };
             };
           };
-
           // this deeplink works only on iOS, make sure it works on android too by using app link
           const deepLinkUrl = new URL(signedKeyRequest.deeplinkUrl);
           const signedKeyRequestToken = deepLinkUrl.searchParams.get("token");
           const signerApprovalUrl = new URL(
-            "https://client.warpcast.com/deeplinks/signed-key-request"
```

Comment:
i think this was changed for android support, so should leave as is

Reply:
added back